development/build/scripts.js

```diff
@@ -306,36 +329,58 @@ function createFactoredBuild({
       const commonSet = sizeGroupMap.get('common');
       // create entry points for each file
       for (const [groupLabel, groupSet] of sizeGroupMap.entries()) {
-        // skip "common" group, they are added tp all other groups
+        // skip "common" group, they are added to all other groups
         if (groupSet === commonSet) continue;
 
         switch (groupLabel) {
           case 'ui': {
-            renderHtmlFile('popup', groupSet, commonSet, browserPlatforms);
-            renderHtmlFile(
-              'notification',
+            renderHtmlFile({
+              htmlName: 'popup',
               groupSet,
               commonSet,
               browserPlatforms,
-            );
-            renderHtmlFile('home', groupSet, commonSet, browserPlatforms);
+              useLavamoat: false,
```

Comment:
It would be nice to pass in `useLavamoat` as a CLI flag

Reply:
im not sure the usecase, can add later. would need to specify it for every entry/webpage

Reply:
I mean so the Lavamoat build can be tested without modifying this file. You'd have to go through and swap every `useLavamoat: false,` to `useLavamoat: true,` right?

Reply:
makes sense